src/index.html

```diff
@@ -11,7 +11,115 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080/" 
+    method="post" 
+    autocomplete="off">
+      <fieldset class="form-field">
+        <legend>Personal information:</legend> 
+        <label class="form-field">
+          Surname:
+          <input type="text" name="Surname:" 
+          required
+          >
+        </label>
+        <label class="form-field">
+          Name:
+          <input type="text" name="Name" 
+          required
+          >
+        </label>
```

Comment:
```suggestion
        <label class="form-field">
          Surname:
          <input
            type="text"
            name="Surname:" 
            required
          >
        </label>
        <label class="form-field">
          Name:
          <input
            type="text"
            name="Name" 
            required
          >
        </label>
```
fix formatting everywhere